cmd/create/oidcconfig/cmd.go

```diff
@@ -294,23 +294,31 @@ func run(cmd *cobra.Command, _ []string) {
 		r.Reporter.Errorf("%s", err)
 		os.Exit(1)
 	}
-	oidcConfigStrategy.execute(r)
+	oidcConfigId := oidcConfigStrategy.execute(r)
 	if !args.rawFiles {
 		arguments.DisableRegionDeprecationWarning = true // disable region deprecation warning
-		oidcprovider.Cmd.Run(oidcprovider.Cmd, []string{"", mode, oidcConfigInput.IssuerUrl})
+		providerArgs := []string{"", mode, oidcConfigInput.IssuerUrl}
+		if oidcConfigId != "" {
+			providerArgs = append(providerArgs, "--oidc-config-id", oidcConfigId)
+		} else {
+			r.Reporter.Errorf("Unable to attempt creation of OIDC provider; oidc config ID"+
```

Comment:
I think you need to identify the strategy that the user requested, if it is manual then you can't say it won't be able :thinking: even though the id will still be emtpy